versioned_docs/version-3.2.x/client-libraries-java-setup.md

```diff
@@ -42,10 +42,53 @@ If you use Gradle, add the following information to the `build.gradle` file.
 def pulsarVersion = '@pulsar:version@'
 
 dependencies {
-    compile group: 'org.apache.pulsar', name: 'pulsar-client', version: pulsarVersion
+    implementation "org.apache.pulsar:pulsar-client:${pulsarVersion}"
 }
 ```
 
+### Pulsar BOM
```

Comment:
I did not see an easy way to not violate DRY principle here and factor this section out and use it across main and 3.2x versioned docs. Even if we Markdown supported including other Markdown files (which I dont think it does) we would still have the cross-version to think about. For now it is duplicated in 2 places.

Reply:
Yes, the solution isn't "clean" in that sense.